[Python][Flaky Test] test_tls_with_self_signed_certificate_succeeds

### Test Name

TestTlsCertificates.test_tls_with_self_signed_certificate_succeeds

### Failure Permlink
https://github.com/valkey-io/valkey-glide/actions/runs/22535380115/job/65281692014#step:9:6280

### System Information

aarch64-unknown-linux-gnu

### Language and Version

Valkey 7.0

### Engine Version

Python 3.9

### Logs

```
=================================== FAILURES ===================================
_ TestTlsCertificates.test_tls_with_self_signed_certificate_succeeds[trio-ProtocolVersion.RESP2-True] _

self = <test_tls_certificates.TestTlsCertificates object at 0xea8bf8327b50>
request = <FixtureRequest for <Function test_tls_with_self_signed_certificate_succeeds[trio-ProtocolVersion.RESP2-True]>>
cluster_mode = True, protocol = <ProtocolVersion.RESP2: 1>

    @pytest.mark.parametrize("cluster_mode", [True, False])
    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
    async def test_tls_with_self_signed_certificate_succeeds(
        self, request, cluster_mode: bool, protocol: ProtocolVersion
    ):
        """
        Verify that connection succeeds with custom root certificates.
        """
        ca_cert = get_ca_certificate()
        tls_config = TlsAdvancedConfiguration(root_pem_cacerts=ca_cert)
    
        # Get the TLS cluster addresses
        valkey_cluster = (
            pytest.valkey_tls_cluster if cluster_mode else pytest.standalone_tls_cluster  # type: ignore
        )
    
        client: Union[GlideClient, GlideClusterClient]
        if cluster_mode:
            cluster_advanced_config = AdvancedGlideClusterClientConfiguration(
                tls_config=tls_config
            )
            cluster_config = GlideClusterClientConfiguration(
                addresses=valkey_cluster.nodes_addr,
                use_tls=True,
                protocol=protocol,
                advanced_config=cluster_advanced_config,
            )
            client = await create_client_with_retry(cluster_config)
        else:
            standalone_advanced_config = AdvancedGlideClientConfiguration(
                tls_config=tls_config
            )
            standalone_config = GlideClientConfiguration(
                addresses=valkey_cluster.nodes_addr,
                use_tls=True,
                protocol=protocol,
                advanced_config=standalone_advanced_config,
            )
            client = await create_client_with_retry(standalone_config)
    
        try:
>           result = await client.ping()

tests/async_tests/test_tls_certificates.py:103: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
glide-async/python/glide/async_commands/cluster_commands.py:306: in ping
    bytes, await self._execute_command(RequestType.Ping, argument, route)
glide-async/python/glide/glide_client.py:446: in _execute_command
    return await self._write_request_await_response(request)
glide-async/python/glide/glide_client.py:641: in _write_request_await_response
    return response_future.result()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <glide.glide_client._CompatFuture object at 0xea8bf814da90>

    def result(self) -> Any:
        if self._exception:
>           raise self._exception
E           glide_shared.exceptions.TimeoutError: timed out

glide-async/python/glide/glide_client.py:110: TimeoutError
```

### Glide Version

main

--- a/python/tests/async_tests/test_tls.py
+++ b/python/tests/async_tests/test_tls.py
@@ -61,6 +61,7 @@ class TestTls:
             protocol=protocol,
             use_tls=True,
             root_pem_cacerts=certificate,
+            request_timeout=5000,
         )
 
         await assert_connected(client)

